package.json

```diff
@@ -2,14 +2,15 @@
   "name": "svg-pathdata",
   "version": "6.0.3",
   "description": "Manipulate SVG path data (path[d] attribute content) simply and efficiently.",
-  "main": "lib/SVGPathData.cjs",
-  "module": "lib/SVGPathData.module.js",
+  "main": "./lib/SVGPathData.cjs",
+  "module": "./lib/SVGPathData.module.js",
   "exports": {
+    "types": "./lib/SVGPathData.d.ts",
```

Comment:
Doesn't work without this. It's [really hidden](https://www.typescriptlang.org/docs/handbook/esm-node.html#packagejson-exports-imports-and-self-referencing).